rbac/rbac/urls.py

```diff
@@ -38,7 +39,8 @@
     re_path(r"^{}v1/".format(API_PATH_PREFIX), include("api.urls")),
     re_path(r"^{}v1/".format(API_PATH_PREFIX), include("management.urls")),
     re_path(r"^_private/", include("internal.urls")),
-    path("", include("django_prometheus.urls")),
+    re_path(r"", include("django_prometheus.urls")),
+    path(r"{}ht/".format(API_PATH_PREFIX), include("health_check.urls")),
```

Comment:
just curious here, where does `/ht/` come from? I just wouldn't guess `ht` means health check endpoint

also is there an api spec to update with the new endpoint? not sure if thats automatically generated or something with rbac

Reply:
The "/ht/" endpoint was what was suggested from the django-health-check github readme: [https://github.com/revsys/django-health-check/tree/master](https://github.com/revsys/django-health-check/tree/master) but I can change it to "health" if that's more preferable!

Reply:
ok! no i would go with what the docs recommend for sure :) they know more than me heh